app/src/main/kotlin/at/bitfire/davdroid/ui/DebugInfoScreen.kt

```diff
@@ -57,11 +57,13 @@ fun DebugInfoScreen(
     syncDataType: String?,
     cause: Throwable?,
     localResource: String?,
+    canViewResource: Boolean,
```

Comment:
Don't know whether it's good design, bug idea: what if we make `onViewResource` nullable and when it's `null`, it means that the resource can't be viewed?

Reply:
I had that before and like the explicit way much more. It decouples the view and activity much better, since otherwise we need to pass the nullable intent all the way down to the button and then bubble it up through the anonymous functions/function declarations again. The composables will need to "know" about `Intent`, which I don't think is good.